task 6

```diff
@@ -0,0 +1,22 @@
+class Numbers
+  def sum(number, other)
+    number + other
+  end
+  
+  def difference(number, other)
+    number - other
+  end
+
+  def multiplication(number, other)
+    number * other
+  end
+
+  def division(number, other)
+    number / other
+  end
+end
+calculator = Numbers.new
```

Comment:
In example to task 6 you can see next `Input example: ruby yourscript.rb 5 10 sum`. So you need to give argument to this script (two numbers and operation name).